src/pages/components/overview/accessibility-status.mdx

```diff
@@ -0,0 +1,62 @@
+| <Tag type="green">Tested</Tag>          | When all tests pass and there are no violations when running the automated Equal Access Checker on default state, advanced states, and keyboard navigation.                                                         |
+| <Tag type="blue">Partially tested</Tag> | When some tests pass or are removed, they are either combined into existing tests or substituted with new ones and moved to a different file. Skipped tests stay in the file but are clearly marked with test.skip. |
+| <Tag type="teal">Manually tested</Tag>  | When screen reader tests are conducted manually and validated by a human to ensure quality.                                                                                                                         |
+| <Tag>Not available</Tag>                | Usually, with keyboard navigation tests or screen reader, when this type of test is not available for certain components.                                                                                           |
+| <Tag type="purple">Not yet tested</Tag> | Usually, with experimental or unstable components, tests have not been conducted for either automated or manual testing.                                                                                            |
+
+## Accessibility tests
+
+Accessibility AVT (Accessibility Automated Verification Testing) involves the
+use of automated tools and software to assess the accessibility of a digital
+product against established accessibility standards, such as the Web Content
+Accessibility Guidelines (WCAG).
+
+| Test                | Meaning                                                                                                                               | Possible Tags                                                                                                                                               |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default state       | The default state of the component                                                                                                    | <Tag type="green">Tested</Tag><br/><Tag type="purple">Not yet tested</Tag><br/><Tag type="blue">Partially tested</Tag>                                      |
+| Advanced states     | Other states rather than the default state of a component. These could be interactive states of a component or its multiple variants. | <Tag type="green">Tested</Tag><br/><Tag type="purple">Not yet tested</Tag><br/><Tag type="blue">Partially tested</Tag>                                      |
```

Comment:
```suggestion
| Advanced states     | Tests that ensure additional states of the component are accessible. This could be interactive states of a component or its multiple variants. | <Tag type="green">Tested</Tag><br/><Tag type="purple">Not yet tested</Tag><br/><Tag type="blue">Partially tested</Tag>                                      |
```